### ISSUE TITLE
I am getting reputation from Frostwolf Clan while not in AV

### ISSUE THREAD

**Issue by esvedesz** [elysium-project/bug-tracker#436]
This is a strange one with a backstory. During an AV I suddenly received a screen like you get at the end of the battleground, even got the reputation for it. I though that maybe it was over due to lack of players. But no, the battlegound was still running because I saw people running past me. I couldnt move and there was no button to leave battleground. I have decided to relog and use /camp. I logged back into AV where  I stood but outside the raid group. I could move but when I died I couldn't ressurect at the graveyard, only at the corpse like in open world. I couldn't exit through portal and hearthstone was on CD so I wrote a ticket to get me out of there :). A very nice GM teleported me back to Tarren Mill. Then I noticed that I still get reputation from that batteground. I wrote another ticket. GM saw that I tried to relog and also after his instructions I relogged and deleted my WDB folder. Nothing helped, I still get reputation from AV. I attach screenshots from the conversation, not chronological sry :< you can sort them by looking at rep bar ;p

EDIT: Now when I enter AV I get this screen 10 seconds after entering. To get out I use /camp. When I log in I start falling below map around Dalaran and I get ported to Tarren Mill instantly...Still getting rep...

Tl;dr  Plz dont ban.

![wowscrnshot_021418_221251](https://user-images.githubusercontent.com/36477875/36229930-19a31fe0-11d9-11e8-80b0-ce4bfe65b5e5.jpg)
![wowscrnshot_021418_221258](https://user-images.githubusercontent.com/36477875/36229931-19bca8f2-11d9-11e8-9946-ad172e074549.jpg)
![wowscrnshot_021418_221305](https://user-images.githubusercontent.com/36477875/36229932-19d5f4ec-11d9-11e8-8eb9-71e9f2171bc8.jpg)
![wowscrnshot_021418_221307](https://user-images.githubusercontent.com/36477875/36229933-19f60340-11d9-11e8-8ea4-2b5afe510283.jpg)
![wowscrnshot_021418_221620](https://user-images.githubusercontent.com/36477875/36229934-1a15d42c-11d9-11e8-8c90-7a9acae4f49e.jpg)
![wowscrnshot_021418_223417](https://user-images.githubusercontent.com/36477875/36229935-1a300c7a-11d9-11e8-8171-25f014197294.jpg)
![wowscrnshot_021418_223422](https://user-images.githubusercontent.com/36477875/36229936-1a4aa71a-11d9-11e8-805a-3b011127d6e8.jpg)
![wowscrnshot_021418_223426](https://user-images.githubusercontent.com/36477875/36229937-1a64d360-11d9-11e8-9ea6-4931fe173e75.jpg)

**Comment by shenna-elysium** [comment]
Can't reproduce.
